srcpkgs/python-dlib/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'python-dlib'
+pkgname=python-dlib
+version=19.20.0
+revision=1
+wrksrc=dlib-${version}
+build_style=python-module
+hostmakedepends="cmake python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel
+			 $(vopt_if openblas openblas-devel)
+			 $(vopt_if x11 libX11-devel)"
+short_desc="C++ toolkit containing machine learning algorithms and tools (Python2)"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="BSL-1.0"
+homepage="https://pypi.org/project/dlib/"
+distfiles="${PYPI_SITE}/d/dlib/dlib-${version}.tar.gz"
+checksum=a3f5070df590c3f510dde2f4c84b1b20e3ef64a02932ed657da490834ecac7e4
+build_options="openblas x11"
+desc_option_openblas="Enable OpenBlas for performance"
+desc_option_x11="Enable X11 for GUI components"
```

Comment:
option_x11 has a common description, remove this line (but keep `_openblas`

Reply:
Done. Now it's just python3.